test/pkg/compose/templatingdata/templating_data.go

```diff
@@ -0,0 +1,95 @@
+package templatingdata
+
+import (
+	"fmt"
+	"strings"
+	"text/template"
+
+	"k8s.io/klog/v2"
+)
+
+// TemplatingData contains all values needed for templating Composer's Sources & Blueprints,
+// and other templated artifacts within a MicroShift's test harness.
+type TemplatingData struct {
```

Comment:
I'm really happy to see all of this together in 1 place.

Reply:
Oh yes, these env vars are really messy